packages/leancode_lint/README.md

```diff
@@ -8,9 +8,9 @@ An opinionated set of high-quality, robust, and up-to-date lint rules used at Le
 
 1. Add `leancode_lint` and `custom_lint` as a dev dependency in your project's `pubspec.yaml`.
 
-```sh
-dart pub add leancode_lint custom_lint --dev
-```
+   ```sh
+   dart pub add leancode_lint custom_lint --dev
+   ```
```

Comment:
:raised_eyebrow:

Reply:
This makes the code block aligned to other contents of `1.` (which I think looks nicer in the rendered document)
![image](https://github.com/leancodepl/flutter_corelibrary/assets/18120733/26dbbad4-1b80-425b-8a92-db75c799a6e1)
as opposed to
![image](https://github.com/leancodepl/flutter_corelibrary/assets/18120733/3b1815b6-a889-47e7-a05e-8374d6804fff)